harness/determined/common/api/_util.py

```diff
@@ -0,0 +1,52 @@
+import enum
+from typing import Callable, Iterator, TypeVar, Union
+
+from determined.common.api import bindings
+
+
+class PageOpts(str, enum.Enum):
+    single = "1"
+    all = "all"
+
+
+# All paginated response types which want read_paginated() to work with.
+# A Paginated union member must have a .pagination attribute.
+# hint: grep 'pagination: .*v1Pagination.*' bindings.py -B 7 | sed -E -n -e 's/class (.*):/\1/p'
```

Comment:
now I want the bindings generator to automatically produce this list: I am sure developers will be annoyed with having to add their new APIs here manually.

Reply:
I am sure these developers can either add it themselves or get over it.

Reply:
Eh, I suppose it could be added to bindings.py without too much effort.

Reply:
done, added as a second commit.

Reply:
awesome